lib/ansiblelint/__init__.py

```diff
@@ -140,8 +140,10 @@ def matchyaml(self, file, text):
 
 class RulesCollection(object):
 
-    def __init__(self):
+    def __init__(self, rulesdirs=[]):
+        self._update_rulesdirs(rulesdirs)
```

Comment:
I don't really understand why you need a separate method for this — initializing things belongs to init. Plus it's not clear what sort of magic the method does from its name.
I'd keep
```suggestion
        self.rulesdirs = ansiblelint.utils.expand_paths_vars(rulesdirs)
```
because that method is effectively a oneliner and there's no need to abstract one function call with another function call.